src/zepto.js

```diff
@@ -437,9 +437,10 @@ var Zepto = (function() {
 
     ready: function(callback){
       // don't use "interactive" on IE <= 10 (it can fired premature)
+      // pipe any ready callback call over task/event queue (#1235)
```

Comment:
A descriptive commit is enough. If each line had to have a comment with a link to reasoning behind it, each line of code would need to have a comment. Also, this code is not at all convoluted or surprising. A reader would see a setTimeout and conclude: "oh, I guess they're executing the callback in a deferred way. gotcha."